data/provider/src/main/java/tech/pegasys/teku/api/ChainDataProvider.java

```diff
@@ -746,4 +747,42 @@ public SafeFuture<Optional<UInt64>> getFinalizedStateSlot(final UInt64 beforeSlo
         .getLatestAvailableFinalizedState(beforeSlot)
         .thenApply(maybeState -> maybeState.map(BeaconState::getSlot));
   }
+
+  public SafeFuture<Optional<ObjectAndMetaData<List<PendingDeposit>>>> getStatePendingDeposits(
+      final String stateIdParam) {
+    return stateSelectorFactory
+        .createSelectorForStateId(stateIdParam)
+        .getState()
+        .thenApply(this::getPendingDeposits);
+  }
+
+  private Optional<ObjectAndMetaData<List<PendingDeposit>>> getPendingDeposits(
+      final Optional<StateAndMetaData> maybeStateAndMetadata) {
+    if (maybeStateAndMetadata.isPresent()) {
+      if (!maybeStateAndMetadata
+          .get()
+          .getMilestone()
+          .isGreaterThanOrEqualTo(SpecMilestone.ELECTRA)) {
```

Comment:
NIT: I think we need a `isLessThan()` method for our Milestone check... This would read much better as `isLessThan(SpecMilestone.ELECTRA)` (and without negating the whole expression).

Reply:
thats what i looked for first, i guess i could change it